src/guild/config/guildConfigs.ts

```diff
@@ -149,6 +149,18 @@ guildConfigs.set('546414872196415501', {
 			minimumChatPermission: PermissionLevel.Everyone,
 			overrides: [],
 		},
+		autoReact: {
+			channels: [
+				{
+					channel: '744075635114115142', // #meta-feedback
+					reactions: [':yes:546414872196415501', ':no:546414872196415501'],
```

Comment:
Has linking the emojis like this confirmed to be working? I can't remember the format off the top of my head but this looks odd

Reply:
Yeah, I tested it on my end before making the PR. Learned something new when I tried it like this.